core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -208,7 +208,9 @@ describe('Translation Modal Component', () => {
   });
 
   describe('when clicking on the translatable content', () => {
-    let target: HTMLElement;
+    let nonParagraphTarget: HTMLElement;
+    let paragraphTarget: HTMLElement;
+    let mathTarget: HTMLElement;
```

Comment:
Why initialize these below and not immediately here? If so, we can declare as `const` as well.

Reply:
I made `nonParagraphTarget ` and `mathTarget ` as constants but `paragraphTarget` is kept as it is since it is required for two tests and needs to be initialised before those two tests.

Reply:
`paragraphTarget` is used in multiple tests, but it's still only initialized once, right (in the `beforeEach`)? If so, then we can still declare as a `const` and initialize it immediately.